### PR TITLE
Support updating color overlay and stroke layer effects.

### PR DESCRIPTION
**Pull request by shaoshing**
*This PR is compatible with the DS master branch and can be safely merged.*

Added properties conversion for color overlay and stroke, to support updating these two layer effects in DS.

@volfied can you review?

### CODE REVIEW

File: src/lib/layerEffect.js [hunk 1]
```diff
@@ -126,9 +138,19 @@ define(function (require, exports) {
      * @return {object} the PS friendly blend mode object
      */
     var _blendMode = function (mode) {
+        return _enumerableValue("blendMode", mode);
```
Comment (volfied) [inline src/lib/layerEffect.js]: Sugar alert. can callers of _blendMode just call `_enumerableValue("blendMode"...` themselves?

**Comment by volfied** [comment]
Can you melt off the syntactic sugar? Otherwise looks good!

**Comment by shaoshing** [comment]
No more bad sugar :D Back to you @volfied

**Comment by volfied** [comment]
Mangi!

**Pull request merged**